app/routes/signup.tsx

```diff
@@ -135,61 +142,198 @@ export const loader: LoaderFunction = async ({request}) => {
   })
 }
 
+const TEAM_MAP = {
```

Comment:
Why don't we do something similar to this: https://github.com/kentcdodds/remix-kentcdodds/blob/781a1f33612f43514eb2de6f63f3ea61c6740fd0/app/utils/misc.tsx#L26-L30

```suggestion
const TEAM_MAP: Record<Team, string> = {
```

Reply:
Actually, this:

```suggestion
const TEAM_MAP: Record<Team, {
    image: string,
    label: string,
    focusClassName: string,
  }> = {
```

It's mostly just so we make sure we don't have a typo on the team name today or in the future.

Reply:
Yeah, agree. This should be typed.